docs/software/communication/nccl.md

```diff
@@ -18,11 +18,20 @@ While the container engine sets these automatically when using the NCCL hook, th
 
 ```bash
 export NCCL_NET="AWS Libfabric" # (1)!
+export NCCL_CROSS_NIC=1
+export NCCL_NET_FORCE_FLUSH=1
 export NCCL_NET_GDR_LEVEL=PHB # (2)!
+export NCCL_SOCKET_IFNAME=hsn
```

Comment:
NCCL_CROSS_NIC and NCCL_SOCKET_IFNAME are set in the CE hook, and seem safe to recommend for most users (latter anyway seems to be just a sanity setting to avoid using the wrong network).